File: Assignments/Assignment_01/Task_06.java
    public static int[] removeAll(int[] sourse, int size, int element){
        for (int i=0; i<sourse.length; i++){
            if (sourse[i]==element){
                remove(sourse,size,i);
                size -= 1;
                i -= 1;
            }
        }
        return sourse;
    }

File: Assignments/Assignment_01/Spring-23/Task_06.py
# Test 06: Remove all occurrences of a particular element from an array
from Task_05 import remove

def remove_all(source, element):

  i = 0

  while(i<len(source)):
    if source[i] == element:
      remove(source,i)
    else:
      i += 1


  return source